### DISCUSSION TITLE
pyinstaller and emmett application ...

### DISCUSSION THREAD

**Discussion by 1st-github-account** [emmett-framework/emmett#427] [Q&A] [answered]
Hi
How can I compile an emmett-application (pyinstaller)?
With the old emmett-version this was still possible. But the current version uses a command "emmett serve" to start, and not the pure python file. Does anyone have an idea how I could create a compiled standalone file?
Thanks a lot!
Lu

**Comment by gi0baro** [comment, marked as answer]
You can do something like this:

```python
from emmett.asgi.server import run


def main():
    run(
        ("your_package_name", None),
        host="0.0.0.0",
        port=8000,
        workers=1
    )


if __name__ == "__main__":
    main()
```

**Reply by 1st-github-account** [reply]
That's great, it works - 'Heavy'.
Thanks a lot!
Lu
